lib/script_base.rb

```diff
@@ -64,7 +64,13 @@ def run
 
     if config.deflate?
       require 'zlib'
-      stdout.puts Zlib::Deflate.deflate((result.json || result.table).to_json, Zlib::BEST_COMPRESSION)
+      require 'base64'
+      stdout.puts Base64.encode64(
+        Zlib::Deflate.deflate(
+          (result.json || result.table).to_json,
+          Zlib::BEST_COMPRESSION,
```

Comment:
Did `BEST_COMPRESSION` make a significant difference from the defaults?

Reply:
We also have MessagePack if we'd rather avoid the Base64 encoding

Reply:
re: `BEST_COMPRESSION`, I didn't try others
re: messagepack, I will give it a shot! I thought it was a binary format for some reason

Reply:
Just confirmed that message pack is binary, I think it'll run into the same issue that the binary DEFLATE data had, which is that SSM command seems to eat it.

The original reason I chose JSON was [this blog post](https://www.peterbe.com/plog/msgpack-vs-json-with-gzip) that seemed to indicate JSON + GZIP yielded smaller payloads than MSGPACK

Reply:
ah got it, sounds good!